modules/data.land/tests/testthat/test-clip_and_move_raster_file.R

```diff
@@ -0,0 +1,79 @@
+library(testthat)
+library(terra)
+library(sf)
+library(withr) # Add withr
```

Comment:
I encourage removing all of these library calls -- testthat is already loaded by the time it reads this file, and the others look like you're already calling their functions with `::` below. 

(Styles seem to differ here, but I favor skipping library calls in the test file on the theory it gives a better chance of finding issues of the "behaves weirdly if user didn't load terra first" variety)